### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by jcy2704**
# Milestone 4

Create calculator logic files:
- Calculate
- Operate

### CODE REVIEW

File: src/logic/operate.js
Comment (teekaytech) [inline src/logic/operate.js]: Instead of continuously repeating the `Big(numberOne)` and `Big(numberTwo)` expressions, you can create 2 variables to store the `Big` version of `numberOne` and `numberTwo` at the beginning of the  function. e.g.
```
const bigOne =  Big(numberOne);
const bigTwo =  Big(numberTwo);
```
You can therefore reuse the `bigOne` and `bigTwo` variables throughout the function.

You can also use `mod` for modulus operation. e.g.
```
  if (operation === '%') {
    return bigOne.mod(bigTwo).toString();
  }

```

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @jcy2704  :wave:

__You've made laudable progress in this milestone.
However, some changes are required before it can be approved. Kindly check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] You have not deployed your changes. Kindly do so.
- [ ] The `Issues page` link on your Readme is redirected to your Weather App issues page. Please fix it.
- [ ] Kindly check the inline comments below for other required changes.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by christianotieno — APPROVED** [review]
### Milestone Approved 🍾 
Hello @jcy2704  👋  
##
This milestone has now met all the requirements needed for approval. 👏 
Congratulations on the hard work done with the changes and overall, on this milestone. Good job 👍 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy Coding 💻 
#### Christian Otieno (CR) 
**| [Twitter](https://twitter.com/iamchrisotieno) | [Github](https://github.com/christianotieno) | [LinkedIn](https://www.linkedin.com/in/christianotieno/) |**

**Pull request merged**
